Dockerfile

```diff
@@ -15,10 +15,12 @@
 # 2) Create the OB-USP-A image, then build the application
 # > docker build -f Dockerfile -t obuspa:latest --target runner .
 #
-FROM ubuntu:kinetic AS build-env
+#FROM --platform=$BUILDPLATFORM ubuntu AS build-env
+FROM ubuntu AS build-env
```

Comment:
Is this implicitly using "ubuntu:latest" ? I think that this would be better using a fixed version of ubuntu, rather than latest. Using 'latest' means that you can never be sure when it's going to break as it's a moving target.

Reply:
I think otherway, latest image is the most up-to-date one with fixes of all known issues. Another problem I see is that, the tendency of most of the developers are to use the latest hence the old ones remain not tested and maintained well. For an example issue #95